Formula/verapdf-apps.rb

```diff
@@ -0,0 +1,28 @@
+class VerapdfApps < Formula
```

Comment:
One last question: other packagers (e.g. Arch) seem to package this as `verapdf`. Any reason we shouldn't just use that name instead?

Reply:
@carlocab I'm not particularly bothered to be honest, I just thought that if a [`verapdf-library`](https://github.com/veraPDF/veraPDF-library) package is created in the future, there might be a name clash. Up to you, of course.

Reply:
@carlocab Just fished an issue with livecheck. Let me know about the above and then we can get this PR merged, I presume.

Reply:
Is this the same as what you get when you install the software from [this page](https://verapdf.org/software/)? If so, we should probably call it `verapdf`.